llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1623,3 +1661,129 @@ void VPlanTransforms::createInterleaveGroups(
       }
   }
 }
+
+static bool supportedLoad(VPWidenRecipe *R0, VPValue *V, unsigned Idx) {
+  if (auto *W = dyn_cast_or_null<VPWidenLoadRecipe>(V->getDefiningRecipe())) {
+    if (W->getMask())
+      return false;
```

Comment:
```suggestion
    return !W->getMask();
```

Reply:
Dropped the check and return as it is subsumed by the check below, thanks!